City-of-Helsinki/drupal-tools
# UHF-9718: Command to check module versions

## How to install
- `composer require drupal/helfi_drupal_tools:dev-UHF-9718`

## How to test

- Run `drush helfi:tools:check-composer-versions /app/composer.lock`
- Make sure out-of-date packages are listed and return code 3 is returned (`echo $?`)
- Update said packages, rerun the command and make sure exit code 0 is returned
